classes/item.rb

```diff
@@ -1,8 +1,9 @@
+require 'date'
 require_relative './classes/book-store/label'
```

Comment:
I suggest you to change `'require_relative './classes/book-store/label'` to `'require_relative './book-store/label'`. Because they are on same directory.

Reply:
@ZahraArshia  I will fix this one since it is easy 😮‍💨